apisix/schema_def.lua

```diff
@@ -277,20 +277,6 @@ local health_checker = {
                     }
                 }
             },
```

Comment:
```suggestion
            }
```
Seems we can remove it as well.

Reply:
IMHO, I like to keep a comma at the end, so adding a new field doesn't need to touch the original code and break the `git blame`.